### PR TITLE
Add icon to Chart.yml for happa

### PR DESCRIPTION
**Pull request by mogottsch**
For https://github.com/giantswarm/roadmap/issues/1248

### CODE REVIEW

File: helm/cluster-autoscaler-app/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 apiVersion: v1
 appVersion: v1.24.0
 description: A Helm chart for the cluster autoscaler.
+icon: https://s.giantswarm.io/app-icons/k8s-initiator/1/dark.svg
```
Comment (whites11) [inline helm/cluster-autoscaler-app/Chart.yaml:4]: k8s-initiator icon for cluster-autoscaler?

Comment (mogottsch) [reply helm/cluster-autoscaler-app/Chart.yaml:4]: I understand your confusion. We are reusing the existing icons. K8s-initiator is just the Kubernetes icon. 
As far as I understand it, autoscaler is from Kubernetes, right?

If the Kubernetes icon does not fit, I am open to suggestions.

Comment (whites11) [reply helm/cluster-autoscaler-app/Chart.yaml:4]: works for me

**Review by whites11 — APPROVED** [review]

**Comment by mogottsch** [comment]
@whites11 Can I bypass the `basic-integration-test`? It seems like you removed it already, but it still prevents me from merging.

**Comment by whites11** [comment]
yeah just force merge

**Pull request merged**